### PR TITLE
OHRM-65 : Version change based on config value 

### PR DESCRIPTION
**Pull request by RajithaKumara**
improved version updater task and updated version to 4.1.2.

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -210,7 +210,7 @@ <h2>4. How to install the pre-requisities manually on Windows?</h2>
 	<ul>
 		<li>Apache HTTP Server ver 1.33 or later</li>
 		<li>PHP Windows package ver 5.1.0 or later</li>
-		<li>MySQL Server ver 4.1.1 or later</li>
+		<li>MySQL Server ver 4.1 or later</li>
```
[the remark this answers is not in the document]
Comment (RajithaKumara) [reply faq.html]: That change happen when OrangeHRM version changing from 4.1 to 4.1.1. I checked with v4.1 and v4.1.1 tags.

[the remark this answers is not in the document]
Comment (RajithaKumara) [reply faq.html]: I go through v4.0, v4.1, v4.1.1 tags and found the commit you are done. https://github.com/RajithaKumara/orangehrm/commit/2e8ebfa1e8f78290b1a3b467bc05f956fbf9fb06#diff-a5c0357af3f61c824310dc44e100c42a
So I can't make sure it's version change or `Versions Requirements`. If you can check with the commit and let me know.

[the remark this answers is not in the document]
Comment (samanthajayasinghe) [reply faq.html]: @rimaz523 can you check this ?

File: version.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+version: 4.1.2
```
[the remark this answers is not in the document]
Comment (rimaz523) [reply version.yml]: @RajithaKumara move the version to the sysConf.php are requested.

File: devTools/general/update-version.php [hunk 3]
```diff
@@ -1,27 +1,14 @@
...
-    'installer/registrationMessage.php',
-    'symfony/plugins/orangehrmBeaconPlugin/modules/communication/actions/sendBeaconMessageAjaxAction.class.php',
-    'symfony/plugins/orangehrmBeaconPlugin/modules/communication/actions/beaconAboutComponent.class.php',
-    'symfony/plugins/orangehrmBeaconPlugin/lib/service/BeaconCommunicationsService.php'
+    'devTools/installer/SPEC/installer.nsi',
+    'devTools/installer/SPEC/main.nsi',
+    'lib/confs/Conf.php-distribution',
+    'lib/confs/sysConf.php',
+    'installer/guide/index.js',
+    'version.yml',
```
Comment (samanthajayasinghe) [inline devTools/general/update-version.php]: Still need this file ? I guess we move config to SysConfig.php file

**Pull request merged**